lib/cassanity/result_transformers/prepared_statement.rb

```diff
@@ -6,8 +6,8 @@ class PreparedStatement
 
       # Internal: Returns the given result as a Cassanity::PreparedStatement
       # object. Meant to wrap a Cql::Client::PreparedStatement
-      def call(result, args = nil)
-        ::Cassanity::PreparedStatement.new result
+      def call(driver, result, args = nil)
```

Comment:
Rather than include driver for all, seems like we should just pass it as an extra arg. That is what args is for.